xoinvader/ingame.py

```diff
@@ -184,6 +252,10 @@ def events(self):
 
     def update(self):
         self._collision_manager.update()
+        self.wave.update()
+        if not self.wave.running:
```

Comment:
Because I want enemies to appear constantly on the screen, not only once.
It's only for test, of course.